src/NzbDrone.Core/Parser/LanguageParser.cs

```diff
@@ -178,10 +181,13 @@ private static Language RegexLanguage(string title)
 
             if (match.Groups["hebrew"].Success)
                 return Language.Hebrew;
+            
+            if (match.Groups["polish"].Success)
+                return Language.Polish;
 
             if (match.Groups["chinese"].Success)
                 return Language.Chinese;
-
+            
```

Comment:
Nit: Remove the spaces on the blank line.

Reply:
@markus101 All done